### DISCUSSION TITLE
Backend request: Sumo Logic

### DISCUSSION THREAD

**Discussion by arishwani** [SigmaHQ/pySigma-plugin-directory#19] [Ask for Backends]
Hi Team, 
I am having issues adding Sumo logic to list of targets, currently, I don't see it when I run the command:

 sigma list targets

**Comment by arishwani** [comment]
<img width="285" alt="sigma" src="https://user-images.githubusercontent.com/33989965/182402209-d9a5e770-0b97-4373-a1e2-c24b3a11b648.png">

**Comment by thomaspatzke** [comment]
There is no SumoLogic backend yet, but I know one is in development 😉 I keep the issue open until it's available and move it to Sigma CLI.

**Comment by createchange** [comment]
Any updates on this? Would be useful for me, whether it is vanilla Sumo Logic, or the CSE variant.

**Reply by createchange** [reply]
@thomaspatzke Can you comment on the above? Would be super useful for us, as we are looking to move into CSE.

**Reply by thomaspatzke** [reply]
I have no updates on this, unfortunately.

**Reply by createchange** [reply]
Bummer, but thanks for prompt response!
